src/main/java/com/crowdin/cli/commands/picocli/TaskAddSubcommand.java

```diff
@@ -0,0 +1,92 @@
+    private String title;
+
+    @CommandLine.Option(names = {"--type"}, paramLabel = "...", descriptionKey = "crowdin.task.add.type")
+    private String type;
+
+    @CommandLine.Option(names = {"--language"}, paramLabel = "...", descriptionKey = "crowdin.task.add.language")
+    private String language;
+
+    @CommandLine.Option(names = {"--file"}, descriptionKey = "crowdin.task.add.file-id", split = ",")
+    private List<Long> files;
+
+    @CommandLine.Option(names = {"--workflow-step"}, paramLabel = "...", descriptionKey = "crowdin.task.add.workflow-step")
+    private Long workflowStep;
+
+    @CommandLine.Option(names = {"--description"}, paramLabel = "...", descriptionKey = "crowdin.task.add.description")
+    private String description;
+
+    @CommandLine.Option(names = {"--skip-assigned-strings"}, paramLabel = "...", negatable = true, descriptionKey = "crowdin.task.add.skip-assigned-strings")
+    private boolean skipAssignedStrings;
+
+    @CommandLine.Option(names = {"--skip-untranslated-strings"}, paramLabel = "...", negatable = true, descriptionKey = "crowdin.task.add.skip-untranslated-strings")
+    private boolean skipUntranslatedStrings;
+
+    @CommandLine.Option(names = {"--label"}, paramLabel = "...", descriptionKey = "crowdin.task.add.label")
+    private List<Long> labels;
+
+    @Override
+    protected NewAction<ProjectProperties, ClientTask> getAction(Actions actions) {
+        int intType = TRANSLATE_TASK_TYPE.equalsIgnoreCase(type) ? 0 : 1;
+        return actions.taskAdd(title, intType, language, files, workflowStep, description, skipAssignedStrings, skipUntranslatedStrings, labels);
+    }
+
+    @Override
+    protected List<String> checkOptions() {
+        String url = this.getProperties(new PropertiesBuilders(), new PicocliOutputter(System.out, isAnsi())).getBaseUrl();
+        boolean isEnterprise = PropertiesBeanUtils.isOrganization(url);
+        List<String> errors = new ArrayList<>();
+        if (!isEnterprise) {
+            if (Strings.isEmpty(type)) {
+                errors.add(RESOURCE_BUNDLE.getString("error.task.empty_type"));
+            }
+            if (Strings.isNotEmpty(type) && !(TRANSLATE_TASK_TYPE.equalsIgnoreCase(type) || PROOFREAD_TASK_TYPE.equalsIgnoreCase(type))) {
+                errors.add(RESOURCE_BUNDLE.getString("error.task.unsupported.type"));
+            }
+        } else {
+            if (workflowStep == null) {
+                errors.add(RESOURCE_BUNDLE.getString("error.task.empty_workflow_step"));
+            }
+        }
+        if (Strings.isEmpty(title)) {
+            errors.add(RESOURCE_BUNDLE.getString("error.task.empty_title"));
+        }
+        try {
+            if (Strings.isEmpty(language)) {
+                errors.add(RESOURCE_BUNDLE.getString("error.task.empty_language"));
+            } else if (!LocaleUtils.isAvailableLocale(new Locale.Builder().setLanguageTag(language).build())) {
+                throw new IllformedLocaleException();
```

Comment:
It's incorrect to validate language code by using the default `LocaleUtils`. 

As a language value shall be passed Crowdin language code https://developer.crowdin.com/language-codes and this code might be different from the `LocaleUtils` codes for some languages

Reply:
also, the exception name is strange a bit